pkgs/by-name/ko/komikku/package.nix

```
@@ -23,15 +23,15 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "komikku";
-  version = "1.71.0";
+  version = "1.72.0";
   pyproject = false;
 
   src = fetchFromGitea {
     domain = "codeberg.org";
     owner = "valos";
     repo = "Komikku";
     rev = "v${version}";
-    hash = "sha256-Y5mqSXV0a16pZVOgOjIeOrpMh8a53qLRmraLfIhKTVI=";
+    hash = "sha256-ZYnJoTqs8xMEmsVWfLz4s0y5lCXe5kdaEko8QgrDmDg=";
```

```suggestion
    hash = "sha256-Kdt4nEWdxfZB7rmPbCegbj4abfv1nMSvAAC6mmUcv44=";
```
I'm getting different hashes. Maybe a force push?

---

Looks like it, thanks for letting me know.